composer.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "wordpress/twentytwenty",
+  "description": "Twenty Twenty is designed with flexibility at its core.",
+  "type": "package",
+  "license": "GPL-2.0-or-later",
+  "keywords": [
+    "WordPress",
+    "Themes"
+  ],
+  "authors": [
+    {
+      "name": "the WordPress team",
+      "homepage": "https://github.com/WordPress/twentytwenty"
+    }
+  ],
+  "require-dev": {
+      "wptrt/wpthemereview": "^0.2.0",
+      "dealerdirect/phpcodesniffer-composer-installer": "^0.5.0"
+  },
+  "scripts": {
+    "phpcs-i": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcs -i",
+    "lint": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcs -s",
```

Comment:
### Function

The script as-is won't work as the ruleset does not pass the `file`s, so anyone running this locally will get a "_No files to scan_" error.

Also: why pass `-s` to the script ? Either add it to the ruleset or leave it out completely.

### Naming of the scripts

`Linting` is something different from `Sniffing` the code. I'd recommend against calling the `phpcs` script `lint` to prevent confusion. Consider a name like `checkcs` instead.

For `lint-summary`, you could consider `checkcs-summary` (or not adding it at all, people can pass their own extra arguments to composer scripts anyway).

For `format`, an alternative name could be `fixcs`,

### Supported PHP version

Another thing to consider: the `WPThemeReview` ruleset includes `PHPCompatibility` with a `testVersion` of `5.2-`.
What is the minimum PHP version for `TwentyTwenty` ? If it is PHP 5.6, the `testVersion` will need to be overruled in the command-line command, so these composer scripts would be the place to do that.
_Note: The `testVersion` currently can not be overruled from a custom ruleset, this is a known issue in PHPCS._

It might also be helpful to make the minimum PHP version of the theme explicit by setting it in the `require` section of the `composer.json` file.

Reply:
Thanks for the suggestions, I think I just copied over the command names from the [core](https://github.com/WordPress/wordpress-develop/blob/4bd76c15b5f8f311ba93e1bc653cad6f7c297881/composer.json#L18). The `checkcs` and `fixcs` makes more sense definitely 🙂 

I'll add the necessary arguments in the `phpcs.xml.dist` (except the `testVersion`, I'm aware of this issue).
I'll remove the summary, that can be added if needed, as you've mentioned. And I'll use the `lint` for actually linting the php files.

Thanks for the comment :+1: